futures-util/src/stream/buffer_unordered.rs

```diff
@@ -134,16 +134,6 @@ where
     }
 }
 
-impl<St> FusedStream for BufferUnordered<St>
```

Comment:
I think we can keep this-- it should just delegate to the underlying `FuturesUnordered`, right?